sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -66,15 +65,19 @@ export interface LogsColumn {
 export type LogsColumnType = string;
 
 // @public
-export type LogsQueryBatchOptions = OperationOptions;
+export interface LogsQueryBatchOptions extends OperationOptions {
+    throwOnAnyFailure?: boolean;
+}
 
 // @public
 export interface LogsQueryBatchResult {
+    // (undocumented)
+    batchResultStatus: "AllSucceeded" | "AllFailed" | "PartiallySucceeded";
```

Comment:
`"Success" | "Partial Success" | "Failure"` is what I would go with - what do you think?

What are other languages doing here for partial success?

Reply:
Other languages don't have this flag for the overall batch result - they have it for the individual queries

Reply:
I like @maorleger's suggestion to keep the 2 string unions the same. I'm not 100% convinced this is needed at the batchResult level, but do think it should be on each individual result.

I get that if all the results succeeded, you'll know you don't need to check the error for each result, but I expect customers will have a single block that will run on each result, and it will have to check the result status anyway in the case that the `batchResultStatus` is `PartiallySucceeded`. So, I think I'd be in favor of removing it unless we get feedback that it would be useful to have (since we can always add it later.) Especially if we'd be the only language with this otherwise.

Reply:
I like @chradek's suggestion to remove it as well 👍 

I wonder if `AllFailed` case should just throw anyway? That feels like the type of issue we should interrupt the caller for (incorrect config, permissions, etc)

Reply:
I think I will remove this field. Other languages don't have this field.